doc/language_reference/index.rst

```diff
@@ -487,7 +487,9 @@ The return type and parameters of a function are represented by the first and su
 .. doc-check: rflx,session_parameter
 .. code:: ada
 
-   with function Decrypt (Key_Update_Message : Key_Update_Message; Sequence_Number : Sequence_Number; Encrypted_Record : Opaque) return TLS_Inner_Plaintext
+   with function Decrypt (Key_Update_Message : Key_Update_Message;
+                          Sequence_Number    : Sequence_Number;
+                          Encrypted_Record   : Opaque) return TLS_Inner_Plaintext
```

Comment:
I don't really like this formatting because the indentation of the parameters depend on the function name and the return type is on the same line as the last parameter. How about using a similar layout as we decided in https://github.com/Componolit/ada-style/issues/26#issuecomment-875526693?

Reply:
Changed.